### PR TITLE
Setup Hexo for Documentation

### PR DESCRIPTION
**Pull request by theParadoxicalWizard**
Solves #1 
Capturefdsfdg
(Uses Hexo Docs Theme)

### CODE REVIEW

**Comment by theParadoxicalWizard** [comment]
The `index.md` file is currently used as a placeholder..Its same as `readme.md`

**Comment by theParadoxicalWizard** [comment]
@abhishek97 please review.

File: index.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 'use-strict'
 
 module.exports = {
+
+  Controller: require('./lib/controllerFactory'),
```
Comment (abhishek97) [inline index.js]: why?

File: package.json [hunk 2]
```diff
@@ -1,23 +1,55 @@
 {
+
```
Comment (abhishek97) [inline package.json]: I think these are unintended changes, please fix this

**Comment by abhishek97** [comment]
@theParadoxicalWizard I think this PR contains unintended changes. Please fix this. You PR should only contain changes relevant to the issue.

File: index.js (hunk 1 above)
Comment (theParadoxicalWizard) [reply index.js]: seem to have been caused due to a rebase error...resolving them now..

Comment (theParadoxicalWizard) [reply index.js]: created new PR https://github.com/coding-blocks/express-jsonapi-controller/pull/9

**Pull request closed without merging**